packages/manager/modules/pci/src/projects/project/training/jobs/info/info.html

```diff
@@ -189,24 +197,32 @@
                     ></oui-tile-description>
                 </oui-tile-definition>
                 <oui-tile-definition
-                    ng-if="$ctrl.job.spec.resources.mem"
+                    ng-if="$ctrl.job.spec.resources.memory"
                     data-term="{{ 'pci_projects_project_training_jobs_ram' | translate  }}"
                 >
-                    <oui-tile-description
-                        data-ng-bind="$ctrl.job.spec.resources.mem"
-                    ></oui-tile-description>
+                    <oui-tile-description>
+                        {{ $ctrl.job.spec.resources.memory |
```

Comment:
**thought:** For upcoming changes, please avoid using `{{`. Prefer using `ngBind` directive.